lib/deaf_grandma.rb

```diff
@@ -19,9 +19,17 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    upper = /[A-Z]+/
+    lower = /[A-Z]?[a-z]+/
+    bye = /\A(BYE ?){3}\z/
+    response = ''
+    if lower.match(input)
+      response = 'SPEAK UP SONNY!'
+    elsif bye.match(input)
+      response = 'SEE YOU LATER SONNY!'
+    elsif upper.match(input)
```

Comment:
What if the input matches none of these regular expressions? I think that would happen if there is no letter in the input.